library/intel/adi_jesd204/adi_jesd204_hw.tcl

```diff
@@ -235,6 +235,15 @@ proc create_lane_pll {id tx_or_rx_n pllclk_frequency refclk_frequency num_lanes
   } elseif {$device_family == "Stratix 10"} {
     add_instance lane_pll altera_xcvr_atx_pll_s10_htile
     set_instance_parameter_value lane_pll {rcfg_enable} {1}
+    set_instance_parameter_value lane_pll enable_mcgb {true}
+    if {$bonding_clocks_en} {
+        set_instance_parameter_value lane_pll {enable_bonding_clks} {true}
+        set_instance_parameter_value lane_pll {mcgb_div} {1}
+        set_instance_parameter_value lane_pll {set_ref_clk_div} {1}
+        set_instance_parameter_value lane_pll {pma_width} {40}
+    } else {
+        set_instance_parameter_value lane_pll enable_hfreq_clk {true}
+    }
```

Comment:
You don't seems to check here if the number of lanes is greater than 6.  So mcgb will be always enabled, even for less lanes ?